other/materials_designer/workflows/analyze_convex_hull.ipynb

```
@@ -0,0 +1,404 @@
+   "execution_count": null,
+   "id": "11",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Resolve material set (if specified)\n",
+    "set_id = None\n",
+    "if MATERIAL_SET:\n",
+    "    set_query = {\"owner._id\": ACCOUNT_ID, \"isEntitySet\": True,\n",
+    "                 \"name\": {\"$regex\": MATERIAL_SET, \"$options\": \"i\"}}\n",
+    "    sets = client.materials.list(set_query)\n",
+    "    if not sets:\n",
```

Ruff (0.15.20)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)